### DISCUSSION TITLE
PrimeNG Ultima Template Angular 18-19 Support

### DISCUSSION THREAD

**Discussion by nazrulcsebd** [primefaces/community#3235] [PrimeNG]
Are there any plans to upgrade the **Ultima** template based on **Angular 18 or 19**? I can not find any discussion on **Angular 18 or 19** and **Ultima**. 

We are currently upgrading our project to 18 version but we can't upgrade to **PRIMING 18** as **Ultima** does not update yet. Can you please give us a timeline, how long it needs to release the 18/19 version of **Ultima**?

**Comment by Tia003** [comment]
Premium Templates:
Application templates like Anima do not support PrimeNG v18 yet, all templates will be updated to PrimeNG v18 by the end of the 2024.

[Link to PrimeNg website on update news page](https://primeng.org/guides/migration#breaking)

**Comment by johnhwright** [comment]
I'm not seeing the updated template in the store for v18 or v19, has this not been completed yet?
